sound/soc/sof/intel/hda-trace.c

```diff
@@ -75,7 +76,6 @@ int hda_dsp_trace_release(struct snd_sof_dev *sdev)
 
 	if (sdev->hda->dtrace_stream) {
 		hstream = &sdev->hda->dtrace_stream->hstream;
-		hstream->opened = false;
 		hda_dsp_stream_put_cstream(sdev,
```

Comment:
why is this removed? Does this belong in this patch?

Reply:
this is actually fixing another issue I met. Let me split it.